templates/puppet4.conf.j2

```diff
@@ -14,6 +14,12 @@
     environment = {{ puppet_environment }}
     environmentpath = $codedir/environments
 
+		{% if puppetize_srv_domain %}
+		use_srv_records = true
```

Comment:
Hello!

Thank you for this PR.

Lately #15  we've been making changes to this role. For example we moved the environment & environmentpath settings down to [agent] section because that made the rabbitmq module we use work.

I see now that this PR is from a commit before that change. I hope the new location of those settings work for you? Maybe those srv_settings should also be under [agent] and not under [main]?

I've been thinking maybe we should try to sort out this puppet4.conf template so it's more flexible.
I'll whip up a PR for that too while we are at this.
Also some debian family testing should also be added :)

Reply:
Proposal up in #20 about adding new settings to the puppet config.
I think going forward it would be nice to have this role more flexible rather than keeping up with all the settings one can potentially have in puppet.conf

Reply:
#20 looks good, closing in favor of that one.